### DISCUSSION TITLE
Environment parameter to control data seeding

### DISCUSSION THREAD

**Discussion by strat1971** [ebean-orm/ebean#2891] [General]
Hi Rob
I am migrating an ebean setup from version 11. It was setting a placeholder name value pair in the DbMigrationConfig which would then be passed the environment name to a repeatable migration stored procedure to control which data is inserted for testing. Now that DbMigrationConfig is gone I can't do that. How would you recommend doing that in version 12?

**Comment by rbygrave** [comment]
Refer to discussion at https://github.com/ebean-orm/ebean/issues/2892
